src/main/webapp/css/user_style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  text-align: center;
+}
+
+.heading {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  height: 40vh;
+}
+
+.logo {
+  text-decoration: none;
+  margin: 30px 0px;
+}
+
+h1 {
+  font: 400 60px Roboto, Arial, sans-serif;
+  margin: 0;
+}
+
+.row {
+  max-width: 400px;
+  margin: 25px auto;
+}
+
+.input-websites, input[type=text] {
+  width: 100%;
+  border: 1px solid transparent;
```

Comment:
Having a transparent border seems really weird. Since this is under border-box, it becomes effectively part of the padding.

It looks like all you're trying to do is get rid of the ugly border. In that case, do `border: none`.